src/plays/web3-staking-dapp/artifacts/contracts/Staking.sol/Staking.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is this file needed?

Reply:
I think for a web3 dapp we should keep all the files related to it so I have kept the files